pkg/harness/harness.go

```diff
@@ -1,27 +1,80 @@
 package harness
 
 import (
-	"github.com/giantswarm/micrologger"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
 )
 
-type Harness struct {
-	logger micrologger.Logger
+// Status represents the current state of the
+// test harness. It is the token passed amongst
+// tasks.
+type Status struct {
+	RemoteCluster bool   `yaml:"remoteCluster"`
+	ImageTag      string `yaml:"imageTag"`
+}
```

Comment:
> Status represents the current state

Looks like `State` is more appropriate name.
`ImageTag ` doesn't look like state. It looks like a configuration. Can we extend Task like:

```
type Task func(harness.Cofnig, *harness.Status) (*harness.Status, error)
```

Reply:
Agree, `State` is much better. Btw, in the next PR there are types for each of the packages which are configurable, if you don't mind we can postpone this fix until then.